lib/ld-eventsource/client.rb

```diff
@@ -288,7 +349,8 @@ def log_and_dispatch_error(e, message)
     def build_headers
       h = {
         'Accept' => 'text/event-stream',
-        'Cache-Control' => 'no-cache'
+        'Cache-Control' => 'no-cache',
+        'User-Agent' => 'ruby-eventsource'
```

Comment:
Is there a particular reason you added a default user-agent?

Reply:
No strong reason, beyond having a user agent identifying the library is fairly common.